Guides/NUI/Components: Added namespace info and deprected note (#1274)
Samsung/tizen-docs: docs/application/dotnet/guides/nui/nui-components/Popup.md

![PopupProperty](./media/PopupProperty.PNG)

> [!NOTE]

> > Popup is deprecated in API8 and will be removed in API10.

Reviewer: Global.
How about writing in Tizen Studio version?

Popup is deprecated since Tizen 6.0 and will be removed after two releases.

> ## Add namespace using
> Popup is defined in namespace Tizen.NUI.Components.

Reviewer 2: Please verify the following suggestion for technical accuracy.

```suggestion
## Add namespace
To implement popup, add namespace `using Tizen.NUI.Components` in your application:
```
